src/robot/reporting/jsbuildingcontext.py

```diff
@@ -20,13 +20,16 @@
 from robot.utils import (attribute_escape, get_link_path, html_escape,
                          html_format, is_string, is_unicode, timestamp_to_secs,
                          unic)
+from robot.result.autoexpandkeywordmatcher import AutoExpandKeywordMatcher
 
 from .stringcache import StringCache
 
 
+
 class JsBuildingContext(object):
 
-    def __init__(self, log_path=None, split_log=False, prune_input=False):
+    def __init__(self, log_path=None, split_log=False, prune_input=False,
+                 auto_expand_args=[]):
```

Comment:
Mutable default values are always a bit questionable. Would be better to use `None` or possibly `()` here.

Reply:
changed default to None + added support of None in AutoExpandKeywordMatcher

Reply:
refactoring to None + implemented handling of None